guide/src/examples/without-a-bundler.md

```diff
@@ -35,9 +35,18 @@ Otherwise the rest of the deployment magic happens in `index.html`:
 {{#include ../../../examples/without-a-bundler/index.html}}
 ```
 
+> **Note**: You cannot directly open `index.html` in your web browser due to [CORS][cors]
+> limitations. Instead, you can set up a quick development environment using
+> Python's built-in HTTP server:
+> ```sh
+> wasm-pack build --target web
+> python3 -m http.server 8080
```

Comment:
Would it be possible to suggest some other kind of HTTP server? (One that's based on Node.js or Rust)

As for why I'm asking: Installing Python on Windows can be a bit finicky and does not always lead to a good user experience.

Reply:
I don't see any changes, I'm guessing something got lost during a git command somewhere?

Reply:
Oops thanks for pointing that out; should be fixed now I think

Reply:
Yay, nice